improver/precipitation_type/shower_condition.py

```diff
@@ -32,6 +32,7 @@
 
 import iris
 import numpy as np
+from iris.cube import CubeList
```

Comment:
I don't think you are using this import.

Reply:
Done.